packages/desktop/views/dashboard/governance/views/DetailsView.svelte

```diff
@@ -21,6 +21,7 @@
     import { openPopup } from '@auxiliary/popup'
```

Comment:
Not changes you made, but would be nice to have more specification in the import here.